vm/devices/storage/disk_nvme/nvme_driver/src/queue_pair.rs

```diff
@@ -469,3 +611,51 @@ pub(crate) fn admin_cmd(opcode: spec::AdminOpcode) -> spec::Command {
         ..FromZeroes::new_zeroed()
     }
 }
+
+#[repr(C)]
+#[derive(Protobuf, Clone, Debug, AsBytes, FromBytes, FromZeroes)]
+#[mesh(package = "underhill")]
+pub struct PendingCommandSavedState {
```

Comment:
Please move all the saved state structs into a single module. This way, we can easily detect and review changes to the saved state in the future.

Reply:
will do